include/ibamr/AdvDiffConservativeMassScalarTransportRKIntegrator.h

```diff
@@ -0,0 +1,312 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (c) 2018 - 2020 by the IBAMR developers
+// All rights reserved.
+//
+// This file is part of IBAMR.
+//
+// IBAMR is free software and is distributed under the 3-clause BSD
+// license. The full text of the license can be found in the file
+// COPYRIGHT at the top level directory of IBAMR.
+//
+// ---------------------------------------------------------------------
+
+/////////////////////////////// INCLUDE GUARD ////////////////////////////////
+
+#ifndef included_IBAMR_AdvDiffConservativeMassScalarTransportRKIntegrator
+#define included_IBAMR_AdvDiffConservativeMassScalarTransportRKIntegrator
+
+/////////////////////////////// INCLUDES /////////////////////////////////////
+#include "ibamr/STSMassFluxIntegrator.h"
```

Comment:
```suggestion
#include <ibamr/config.h>

#include "ibamr/STSMassFluxIntegrator.h"
```
I need to finish the CI check to verify that the config header is always present - we always need it.